client/mass/groups.js

```diff
@@ -143,7 +148,9 @@ class MassGroups {
 		if (this.state.customTerms.length == 0) {
 			this.dom.customTermDiv
 				.append('div')
-				.text('No custom variables. Use above controls to create new ones.')
+				.text(
+					'No custom variables. Use above controls to create new ones. Custom variables are added to dictionary terms'
```

Comment:
please replace as `are added to dictionary.`
add missing period at the end
just say dictionary as that's where they appear together with official terms
also on the ui let's print "variable" but not "term"